studio/tartifact_store.py

```diff
@@ -21,11 +21,16 @@
 from .util import compression_to_extension, compression_to_taropt, timeit
 from .util import sixdecode
 
+from .base_artifact_store import BaseArtifactStore
 
-class TartifactStore(object):
+
+class TartifactStore(BaseArtifactStore):
 
```

Comment:
TartifactStore is now a subclass of BaseArtifactStore,
so as to make underlying connection client available in all subclasses.